src/main/java/com/slimebot/games/wordchain/Wordchain.java

```diff
@@ -0,0 +1,179 @@
+                .setDescription("Um zu erfahren wie \"Wortkette\" funktioniert nutze ```/wordchain explanation```") // TODO
+                .addField("Spielleiter:in:", "<@"+gameMaster+">", true)
+                .addField(":timer: Timeout:", seconds + "s", true)
+                .addField(":x: Max Fehler:", lives + " Fehler", true)
+                .addField("Spieler:", players.stream().map(p -> p.getAsMention()).collect(Collectors.joining("\r\n")), true)
+                .setFooter("GameID: " + uuid)
+                .setTimestamp(Instant.now());
+    }
+
+    private void nextTurn() {
+        if(scheduledFuture != null)scheduledFuture.cancel(true);
+
+        if(players.size() == 1) {
+
+            // TODO: Winn stuff
+
+            sendMessage("Spiel vorbei! " + players.get(0).getAsMention() + " hat gewonnen!").queue(); // TODO: better message
+            end();
+            return;
+        }
+
+        round++;
+        if ((playerTurn + 1) < this.players.size()) {
+            playerTurn++;
+        } else playerTurn = 0;
+
+        if(lastWord == null)sendMessage(round + ": " + this.players.get(playerTurn).getAsMention() + " ist an der Reihe! Suche dir ein Wort aus!").queue();
+        if(lastWord != null)sendMessage(round + ": " + this.players.get(playerTurn).getAsMention() + " ist an der Reihe! Das letzte Wort ist \"**" + lastWord + "**\"").queue();
+
+        scheduledFuture = Main.executor.schedule(() -> {
+            damage(this.players.get(playerTurn), (p, l) -> {
+                sendMessage(":x: Die Zeit von "+ p.getAsMention() + " ist abgelaufen und hat nur noch **" + l + "** Versuche!").queue();
+            });
+        }, seconds, TimeUnit.SECONDS);
+    }
+
+    private void damage(WordchainPlayer player, BiConsumer<WordchainPlayer, Short> messageConsumer) {
+        short lives = player.lives;
+        if(lives <= 1) {
+            kickPlayer(player, sendMessage(":x: " + player.getAsMention() +" ist Ausgeschieden!"));
+            nextTurn();
+            return;
+        }
+        lives--;
+        player.lives = lives;
+
+        messageConsumer.accept(player, lives);
+        nextTurn();
+    }
+
+    private void kickPlayer(GamePlayer player, RestAction<?> restAction) {
+        restAction.queue();
+
+        leave(player);
+    }
+
+    public MessageEmbed updateJoinEmbed(MessageEmbed embed) {
+        return new EmbedBuilder()
+                .setColor(Main.embedColor(String.valueOf(guildId)))
+                .setTitle(embed.getTitle())
+                .setDescription("Um zu erfahren wie \"Wortkette\" funktioniert nutze ```/wordchain explanation```") // TODO
+                .addField("Spielleiter:in:", "<@" + gameMaster + ">", true)
+                .addField(":timer: Timeout:", seconds + "s", true)
+                .addField(":x: Max Fehler:", lives + " Fehler", true)
+                .addField("Spieler:", players.stream().map(p -> p.getAsMention()).collect(Collectors.joining("\r\n")), true)
```

Comment:
Create method in Game that returns a player list as string
Just use `\n` without `\r`